src/Sylius/Behat/Client/Helper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+use Sylius\Behat\Service\SharedStorageInterface;
+use Webmozart\Assert\Assert;
+
+final class Helper
+{
+    public function __construct(
+        private SharedStorageInterface $sharedStorage,
+        private RequestBuilderFactoryInterface $requestBuilderFactory,
+        private ApiClientInterface $apiClient,
+        private string $authorizationHeader,
+    ) {
+    }
+
+    public function update(callable $closure): void
+    {
+        /** @var RequestBuilder $requestBuilder */
+        $requestBuilder = $this->sharedStorage->get('request_builder');
```

Comment:
the problem with this implementation is that we need to remember to put request builder in shared storage anyway. Thats quite an assumption, and it is implicit, which may lead to errors or just people will not use it